internal/storage/rocksdb/store.go

```diff
@@ -438,9 +487,21 @@ func (rdbIter *iter) HasNext() bool {
 }
 
 func (rdbIter *iter) Next() ([]byte, []byte) {
-	defer rdbIter.rdbIter.Next()
 	key := toByteArray(rdbIter.rdbIter.Key())
 	val := toByteArray(rdbIter.rdbIter.Value())
+	expireTs, _ := getExpireTs(val)
+	if expireTs > 0 && expireTs < time.Now().Unix() {
+		//	rdb.Delete(key) //trigger delete.
+		rdbIter.rdbIter.Next()
+		if rdbIter.HasNext() {
+			return rdbIter.Next()
+		} else {
```

Comment:
Please remove this `else` and inline its return statement.

Reply:
Done